sources/docker-compose.yml

```diff
@@ -24,6 +24,13 @@ services:
       - ENVIRONMENT=local
       - E2E_TEST_STUDENT_USERNAME=${E2E_TEST_STUDENT_USERNAME}
       - E2E_TEST_STUDENT_PASSWORD=${E2E_TEST_STUDENT_PASSWORD}
+      - E2E_TEST_BCeID_USERNAME=${E2E_TEST_BCeID_USERNAME}
```

Comment:
Same comment as previous ones, on the username and password . Can we have it like some mention of INSTITUTION there

Reply:
Or INSTITUTIONUSER

Reply:
This user/password is not related to an institution. This user could be used to any other E2E test that requires a valid BCeID User.
Under this scenario, do you still believe that we should change it? If yes, I would avoid adding "institution" to it.